e2e/upgrade-cephfs.go

```diff
@@ -128,13 +154,55 @@ var _ = Describe("CephFS Upgrade Testing", func() {
 				if err != nil {
 					e2elog.Failf("failed to load application with error %v", err)
 				}
+				label := map[string]string{
+					"app": appLabel,
```

Comment:
define app as a key and use it when generating the ListOptions.  and also this need to be addressed in other places also

Reply:
In the code, the key "app" has different values for different pods.... For eg. used the label "validate-snap-clone" for validating clone from snapshots and "validate-clone" for a PVC clone and they all have the same key. Are you indicating to use different keys as well?

Reply:
Same key is fine define app as variable and use it when you are generator the listoptions